src/runtime/components/CommandPalette.vue

```diff
@@ -259,7 +260,7 @@ const groups = computed(() => {
       <ComboboxContent :class="ui.content({ class: props.ui?.content })" :dismissable="false">
         <ComboboxEmpty :class="ui.empty({ class: props.ui?.empty })">
           <slot name="empty" :search-term="searchTerm">
-            {{ searchTerm ? `No results for ${searchTerm}` : 'No results' }}
+            {{ searchTerm ? t('ui.commandPalette.noMatch') : t('ui.commandPalette.noData') }}
```

Comment:
Should you not pass the `searchTerm` to `t('ui.commandPalette.noMatch')` here?

Reply:
Why? The user can already see what they typed into the field. I don't think it's necessary to duplicate text that is a few pixels away from the same text.

Of course, if this is a matter of principle, I will add `searchTerm`.

Reply:
You make a point. Maybe we can add it but keep the translation as-is so users have the choice?

Reply:
That's a good idea! In case the user wants to override the translation with his own, he will have the `search` variable available inside the literal